component/vshn_postgres.jsonnet

```diff
@@ -37,6 +37,8 @@ local connectionSecretKeys = [
   'POSTGRESQL_PORT',
   'POSTGRESQL_USER',
   'POSTGRESQL_PASSWORD',
+  'PRIMARY_IP',
```

Comment:
Maybe call it `EXTERNAL_IP` or `LOADBALANCER_IP`, that would make it clearer.

Reply:
I like LOADBALANCER_IP I'll switch name to that